### PR TITLE
Adds a couple wearable pelts to the loadout.

### PR DESCRIPTION
**Pull request by Ketrai**
## About The Pull Request

Adds wearable animal pelts to the loadout.
image
image
image

## How This Contributes To The Skyrat Roleplay Experience

More character customization. Look like a savage! Be more difficult to be taken seriously! Make an outrageous fashion statement!

:cl:
add: Added some new wearable pelts to the loadout
/:cl:

### CODE REVIEW

**Comment by Alanii** [comment]
Should name them.

Felinid pelt, vulpkanin pelt, etc.

🤣

**Comment by Nerev4r** [comment]
Can we get the head part to be togglable like a hood? Maybe move them to the neck slot to accompany?

**Comment by Draganfrukts** [comment]
This is your local spriter  here with sprite advice (and misc)

### You don't really leave the eyes in when skinning an animal as they can't be preserved
image

### The side perspective gets a bit weird here.
image
And I suggest doing something like,
image
Which makes the ears look aligned

### The regular tiger pelt is quite saturated
image
Some hue-shifting and a couple of darker colors and it turns out much nicer

### A couple of pelts are very sleepy and fell into a pile of pillows!
image
Trying to color huge masses of colors without falling into a pillow shading trap can prove itself difficult sometimes. I'd suggest trying something like this
image
_**DISCLAIMER:** This sprite is meant to be a simplified reference._
You can reduce the number of colors used and darken highlights to attain something similar. Obviously, it entails trying to fill in a couple of details - but shouldn't be too hard.

**Comment by Ketrai** [comment]
These are older sprites yeah. I can see about touching them up, but it depends how much time I want to spend on this between all my other little projects.

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 1]
```diff
@@ -113,6 +113,50 @@
 /obj/item/clothing/head/weddingveil
 	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
 
+/obj/item/clothing/head/pelt
+	name = "Bear pelt"
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
	name = "bear pelt"
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 2]
```diff
@@ -113,6 +113,50 @@
...
+	desc = "A luxurious bear pelt, good to keep warm in winter. Or to sleep through winter."
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
	desc = "A luxurious bear pelt, good to keep warm in winter. Or to sleep through it."
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 3]
```diff
@@ -113,6 +113,50 @@
...
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "bearpelt_brown"
+	inhand_icon_state = "bearpelt_brown"
+
+/obj/item/clothing/head/pelt/black
+	icon_state = "bearpelt_black"
+	inhand_icon_state = "bearpelt_black"
+
+/obj/item/clothing/head/pelt/wolfpelt
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
/obj/item/clothing/head/pelt/wolf
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 4]
```diff
@@ -113,6 +113,50 @@
...
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head/pelt_big.dmi'
+	icon_state = "wolfpelt_brown"
+	inhand_icon_state = "wolfpelt_brown"
+
+/obj/item/clothing/head/pelt/wolfpeltblack
+	name = "Wolf pelt"
+	desc = "A fuzzy wolf pelt, demanding respect as a hunter, well if it isn't synthetic or anything at least. Or bought, for all the glory but none of the credit."
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head/pelt_big.dmi'
+	icon_state = "wolfpelt_gray"
+	inhand_icon_state = "wolfpelt_gray"
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
/obj/item/clothing/head/pelt/wolf/black
	icon_state = "wolfpelt_gray"
	inhand_icon_state = "wolfpelt_gray"
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 5]
```diff
@@ -113,6 +113,50 @@
...
+
+/obj/item/clothing/head/pelt/tigerpelt
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
/obj/item/clothing/head/pelt/tiger
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 6]
```diff
@@ -113,6 +113,50 @@
...
+	name = "Shiny tiger pelt"
+	desc = "A vibrant tiger pelt, particularly fabulous."
+	icon_state = "tigerpelt_shiny"
+	inhand_icon_state = "tigerpelt_shiny"
+
+/obj/item/clothing/head/pelt/tigerpeltsnow
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
/obj/item/clothing/head/pelt/snow_tiger
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 7]
```diff
@@ -113,6 +113,50 @@
...
+	name = "Snow tiger pelt"
+	desc = "A pelt of a less vibrant tiger, but rather warm."
+	icon_state = "tigerpelt_snow"
+	inhand_icon_state = "tigerpelt_snow"
+
+/obj/item/clothing/head/pelt/tigerpeltpink
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
/obj/item/clothing/head/pelt/pink_tiger
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 8]
```diff
@@ -113,6 +113,50 @@
...
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "bearpelt_brown"
+	inhand_icon_state = "bearpelt_brown"
+
+/obj/item/clothing/head/pelt/black
+	icon_state = "bearpelt_black"
+	inhand_icon_state = "bearpelt_black"
+
+/obj/item/clothing/head/pelt/wolfpelt
+	name = "Wolf pelt"
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
	name = "wolf pelt"
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 9]
```diff
@@ -113,6 +113,50 @@
...
+	desc = "A fuzzy wolf pelt, demanding respect as a hunter, well if it isn't synthetic or anything at least. Or bought, for all the glory but none of the credit."
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
	desc = "A fuzzy wolf pelt that demands respect as a hunter... assuming it wasn't just purchased, that is, for all the glory but none of the credit."
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 10]
```diff
@@ -113,6 +113,50 @@
...
+
+/obj/item/clothing/head/pelt/wolfpeltblack
+	name = "Wolf pelt"
+	desc = "A fuzzy wolf pelt, demanding respect as a hunter, well if it isn't synthetic or anything at least. Or bought, for all the glory but none of the credit."
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head/pelt_big.dmi'
+	icon_state = "wolfpelt_gray"
+	inhand_icon_state = "wolfpelt_gray"
+
+/obj/item/clothing/head/pelt/tigerpelt
+	name = "Shiny tiger pelt"
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
	name = "shiny tiger pelt"
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 11]
```diff
@@ -113,6 +113,50 @@
...
+	desc = "A vibrant tiger pelt, particularly fabulous."
+	icon_state = "tigerpelt_shiny"
+	inhand_icon_state = "tigerpelt_shiny"
+
+/obj/item/clothing/head/pelt/tigerpeltsnow
+	name = "Snow tiger pelt"
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
	name = "snow tiger pelt"
```

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 12]
```diff
@@ -113,6 +113,50 @@
...
+	desc = "A pelt of a less vibrant tiger, but rather warm."
+	icon_state = "tigerpelt_snow"
+	inhand_icon_state = "tigerpelt_snow"
+
+/obj/item/clothing/head/pelt/tigerpeltpink
+	name = "Pink tiger pelt"
```
Comment (tf-4) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]:
```suggestion
	name = "pink tiger pelt"
```

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm [hunk 13]
```diff
@@ -224,6 +224,34 @@ GLOBAL_LIST_INIT(loadout_helmets, generate_loadout_items(/datum/loadout_item/hea
...
+	name = "Shiny tiger pelt"
+	item_path = /obj/item/clothing/head/pelt/tigerpelt
+
+/datum/loadout_item/head/tigerpeltsnow
+	name = "Snow tiger pelt"
+	item_path = /obj/item/clothing/head/pelt/tigerpeltsnow
+
+/datum/loadout_item/head/tigerpeltpink
+	name = "Pink tiger pelt"
+	item_path = /obj/item/clothing/head/pelt/tigerpeltpink
```
Comment (tf-4) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm]:
```suggestion
/datum/loadout_item/head/bearpelt
	name = "Brown Bear Pelt"
	item_path = /obj/item/clothing/head/pelt

/datum/loadout_item/head/bearpeltblack
	name = "Black Bear Pelt"
	item_path = /obj/item/clothing/head/pelt/black

/datum/loadout_item/head/wolfpelt
	name = "Brown Wolf Pelt"
	item_path = /obj/item/clothing/head/pelt/wolfpelt

/datum/loadout_item/head/wolfpeltblack
	name = "Black Wolf Pelt"
	item_path = /obj/item/clothing/head/pelt/wolfpeltblack

/datum/loadout_item/head/tigerpelt
	name = "Shiny Tiger Pelt"
	item_path = /obj/item/clothing/head/pelt/tigerpelt

/datum/loadout_item/head/tigerpeltsnow
	name = "Snow Tiger Pelt"
	item_path = /obj/item/clothing/head/pelt/tigerpeltsnow

/datum/loadout_item/head/tigerpeltpink
	name = "Pink Tiger Pelt"
	item_path = /obj/item/clothing/head/pelt/tigerpeltpink
```
update paths here too

**Review by tf-4 — CHANGES_REQUESTED** [review]

**Comment by theselfish** [comment]
Considering these mention winter. Shouldn't they have some cold protection?

Also, I'd like to say while I agree with the rest of Dragon's post. I say keep the eyes. Rule of Cool and such.

**Comment by Ketrai** [comment]
image

Updated bear and tiger sprites. Also fixes the wolf sprite side state ears. Eyes remain, I just like that aesthetic more. I unno, maybe the hunter put in fake eyes for effect!

**Review by tf-4 — APPROVED** [review]

**Pull request merged**